manifests/r/ranpha/LavFiltersMegamix/0.77.1-1/ranpha.LavFiltersMegamix.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+PackageIdentifier: ranpha.LavFiltersMegamix
+PackageVersion: 0.77.1-1
+PackageLocale: en-US
+Publisher: Hendrik Leppkes
+PackageUrl: https://anime.my/tutorials/watching-h264-videos-using-compute-unified-device-architecture-cuda/
+PublisherUrl: https://anime.my
+PublisherSupportUrl: https://anime.my/misc/contact-form/
+PrivacyUrl: https://anime.my/misc/privacy-policy/
+Author: ranpha
+PackageName: LAV Filters 0.77.1-1
+PackageUrl: https://anime.my/tutorials/watching-h264-videos-using-compute-unified-device-architecture-cuda/
```

Comment:
```suggestion
```

Reply:
Apologies, I did not see the PackageUrl already included above. Validation is failing because of this duplicate value.
> Manifest Error: Duplicate field found in the manifest. Field: PackageUrl